src/packets/parsers/result.ts

```diff
@@ -97,9 +98,10 @@ function convertType(field: FieldInfo, val: string): any {
     case MYSQL_TYPE_TINY:
     case MYSQL_TYPE_SHORT:
     case MYSQL_TYPE_LONG:
-    case MYSQL_TYPE_LONGLONG:
     case MYSQL_TYPE_INT24:
       return parseInt(val);
```

Comment:
@magichim

This PR introduces the failure of the [`testPool`](https://github.com/manyuanrong/deno_mysql/blob/b18da6cdfb692e3f9fc36e9d0efe3302020b3f3b/test.ts#L130-L145).

The cause is that `select 1` returns a js `BigInt` instead of `Number`.

What do you think of the following modifications to fix this problem?

```suggestion
      if (val > Number.MAX_SAFE_INTEGER) {
        return BigInt(val);
      } else {
        return parseInt(val);
      }
```

Reply:
I think It's good case.

Reply:
@uki00a  The is another question, your test.ts file is working exactly? My deno version is 1.0 but Deno.test's fn function is not working. In terminal, type "deno run -A test.ts." just work createTestDB function case. How about you?

Reply:
@magichim 
> The is another question, your test.ts file is working exactly?
> In terminal, type "deno run -A test.ts." just work createTestDB function case. How about you?

Yes. In my local environment(deno 1.0.0), all tests pass now.

You need to run `deno test -A` instead of `deno run -A test.ts`.

Reply:
@magichim 
I've fixed the problem at #47.
Please review it again🙏

Reply:
@uki00a 
Oh! really thank you. Deno command is changed haha...
Anyway above your code is MYSQL_TYPE_LONGLONG case?

Reply:
@magichim 

> Anyway above your code is MYSQL_TYPE_LONGLONG case?

No. That's the MYSQL_TYPE_LONGLONG case.

> How about removing decimal test code? It's unsafety.

That problem has been fixed at #47.

Reply:
@uki00a 

Looks good~!!

OKay! Try to merge again! Thanks!